fft/src/KokkosFFT_ROCM_plans.hpp

```diff
@@ -129,8 +127,13 @@ auto create_plan(const ExecutionSpace& exec_space,
       convert_int_type_and_reverse<int, std::size_t>(fft_extents);
 
   // Create the description
-  rocfft_plan_description description;
-  rocfft_status status = rocfft_plan_description_create(&description);
+  std::unique_ptr<rocfft_plan_description,
+                  std::function<void(rocfft_plan_description*)>> const
+      description(new rocfft_plan_description,
+                  [](rocfft_plan_description* desc) {
+                    rocfft_plan_description_destroy(*desc);
+                  });
```

Comment:
```suggestion
                  [](rocfft_plan_description* desc) {
                    rocfft_plan_description_destroy(*desc);
                    delete desc;
                  });
```

Reply:
I introduced a wrapper for description as well for RAII. This part has been removed